testcases/OpTestSTB.py

```diff
@@ -0,0 +1,623 @@
+            print BMC_CONST.CAPP_PAR_NOT_VERIFIED
+            return BMC_CONST.CAPP_PAR_NOT_VERIFIED
+
+        l_msg = "CAPP partition verify details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects BOOTKERNEL verified information from OPAL
+    # messagelog and if found determines whether it is 'verified' or
+    # 'not verified'.
+    # Returns BOOTKERNEL_PAR_VERIFIED for BOOTKERNEL partition verified
+    # Returns BOOTKERNEL_PAR_NOT_VERIFIED for BOOTKERNEL partition not verified
+    # Raises OpTestError when no information found
+    #
+    def bootkernel_verified(self):
+        l_bk_verified = self.string_get(self.l_data,
+                                        BMC_CONST.BOOTKERNEL_VERIFIED)
+        l_bk_not_verified = self.string_get(self.l_data,
+                                            BMC_CONST.SB_SKIPPED)
+        if l_bk_verified:
+            print BMC_CONST.BOOTKERNEL_PAR_VERIFIED
+            return BMC_CONST.BOOTKERNEL_PAR_VERIFIED
+        elif l_bk_not_verified:
+            print BMC_CONST.BOOTKERNEL_PAR_NOT_VERIFIED
+            return BMC_CONST.BOOTKERNEL_PAR_NOT_VERIFIED
+
+        l_msg = "BOOTKERNEL partition verify details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP loaded information from OPAL
+    # messagelog and if found determines whether it is 'loaded' or
+    # 'not loaded'.
+    # Returns CAPP_PAR_LOADED for CAPP partition loaded
+    # Returns CAPP_PAR_NOT_LOADED for CAPP partition not loaded
+    # Raises OpTestError when no information found
+    #
+    def capp_loaded(self):
+        l_capp_loaded = self.string_get(self.l_data,
+                                        BMC_CONST.CAPP_LOADED)
+        if l_capp_loaded:
+            l_capp_loaded_status = self.status_finder(l_capp_loaded,
+                                                      BMC_CONST.LOADED,
+                                                      BMC_CONST.NOT_LOADED)
+            if l_capp_loaded_status == BMC_CONST.LOADED:
+                print BMC_CONST.CAPP_PAR_LOADED
+                return BMC_CONST.CAPP_PAR_LOADED
+            elif l_capp_loaded_status == BMC_CONST.NOT_LOADED:
+                print BMC_CONST.CAPP_PAR_NOT_LOADED
+                return BMC_CONST.CAPP_PAR_NOT_LOADED
+            else:
+                print "CAPP partition loaded related error occured"
+                return BMC_CONST.FW_FAILED
+        else:
+            l_msg = "CAPP partition loaded details not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects BOOTKERNEL loaded information from OPAL
+    # messagelog and if found determines whether it is 'loaded' or
+    # 'loaded'.
+    # Returns BOOTKERNEL_PAR_LOADED for BOOTKERNEL partition loaded
+    # Returns BOOTKERNEL_PAR_NOT_LOADED for BOOTKERNEL partition not loaded
+    # Raises OpTestError when no information found
+    #
+    def bootkernel_loaded(self):
```

Comment:
Change function name to opal_get_bootkernel_loaded_info. and just return True/ False for loaded/not-loaded cases.